### PR TITLE
fix(aio): googlebot polyfill fix

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

**Comment by IgorMinar** [comment]
I tested this setup in webmaster tools and it seems to be finally working.

File: aio/src/index.html [hunk 1]
```diff
@@ -37,20 +37,28 @@
...
-     */
-    function polyfillBrowser(polyfillUrl) {
-      var script = document.createElement('script');
-      script.src = polyfillUrl;
-      document.head.appendChild(script);
-    }
     if (window.document.documentMode) {
-      // polyfill IE11
-      polyfillBrowser('generated/ie-polyfills.min.js');
+      // polyfill IE11 in a blocking way
```
Comment (gkalpak) [inline aio/src/index.html]: The "in a blocking way" part is [debatable][1] :wink:

[1]: https://www.html5rocks.com/en/tutorials/speed/script-loading/#toc-dom-rescue

Comment (IgorMinar) [inline aio/src/index.html]: why? I thought that this was tested on IE. I haven't tested it myself, but I think Pete did.

Comment (gkalpak) [inline aio/src/index.html]: @petebacondarwin used `async = false` iirc. But maybe IE deviates from the spec and this isn't necessary.

Comment (gkalpak) [inline aio/src/index.html]: For reference, here is a [non-conclusive discussion][1] we had about this :smiley:

[1]: https://github.com/angular/angular/commit/be9e8b99ff43cec99ef6f5e5d55558c5e285e627#commitcomment-22205884

**Review by gkalpak — APPROVED** [review]

**Pull request merged**